<ui version="4.0" >
 <class>AccountInformerTest</class>
 <widget class="QWidget" name="AccountInformerTest" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" >
    <widget class="QPushButton" name="addButton" >
     <property name="text" >
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QPushButton" name="deleteButton" >
     <property name="text" >
      <string>Delete</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="AccountInformerView" name="treeView" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AccountInformerView</class>
   <extends>QTreeView</extends>
   <header>accountinformerview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
